DOCUMENTATION.md

```diff
@@ -95,7 +95,7 @@ to be able to process the message.
 
 #### 1.3.4 Private message executor
 
-Some functionality is able when you use private messages instead of adding the Bot to a group.
+There is some functionality that is available when you use private messages instead of when you the Bot to a group.
```

Comment:
`when you add` probably?

Reply:
YEZ. THAT SENTENCE BROKE MY PARSER BIP BLOP